templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,18 @@
+{{#hero}}
+<section class="commerce-hero-content center">
+  {{#images}}
+    {{#image}}
+      <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="responsive" media="{{media}}" alt="{{alt}}"></amp-img>
+    {{/image}}
+  {{/images}}
+  <div class="commerce-hero-content-content commerce-hero-content-theme-{{text-theme}} flex justify-center flex-column">
+    <div>
```

Comment:
As discussed, will leave this in for now since it's there for flexbox positioning.